shared-ui-compose/src/main/kotlin/team/duckie/app/android/shared/ui/compose/UserFollowingLayout.kt

```diff
@@ -78,19 +79,21 @@ fun UserFollowingLayout(
             .height(56.dp)
             .padding(
                 vertical = 12.dp,
-            ),
+            )
+            .quackClickable(
```

Comment:
argument가 하나일 때 줄바꿈 안하는 편이 가독성이 좋을 것 같습니다!
```suggestion
            .padding(vertical = 12)
            .quackClickable(rippleEnabled = false)
```